### PR TITLE
TRIVIAL: remove fossa unrelated requirements

### PR DESCRIPTION
**Pull request by hkad98**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: .fossa/prepare_and_populate_pipenv.sh [hunk 1]
```diff
@@ -15,26 +15,31 @@ cd $PROJ_HOME
 
 cd ./gooddata-api-client
 pipenv install --skip-lock -r requirements.txt
+rm -rf test-requirements.txt type-requirements.txt
```
Comment (helobinvn) [inline .fossa/prepare_and_populate_pipenv.sh:18]:
```suggestion
rm -f test-requirements.txt type-requirements.txt
```

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by helobinvn — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR369

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/181/ : FAILURE in 1m 12s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/331/ : SUCCESS in 1m 05s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/334/ : SUCCESS in 9s

**Pull request merged**